### ISSUE TITLE
OpenGL 2 Jagged borders

### ISSUE THREAD

**Issue by Zorrr** [ocornut/imgui#2026]
**Version/Branch of Dear ImGui:** v1.63 WIP

I implemented ImGui using the following backends:
Opengl2
Modified GLFW backend to fit everything to work using a HWND paramater, and to be able to be drawn in a swapBuffers function.

**My Issue/Question:**
My error lies in the rendering of the frames, my frame borders look jagged and aren't smooth at all (including the arrows), buttons and everything not rounded is not affected as far as I've tested and seen.

This is my current code:
**NOTE:** drawn in a swapbuffers hook
```
                ImGui_ImplOpenGL2_NewFrame ( );
		ImGui_ImplCustom_NewFrame ( );


		ImGui::Begin ( "Hello", NULL, ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoSavedSettings );
		{
			ImGui::Button ( "Hello!" );
			ImGui::End ( );
		}

		ImGui::Render ( );
		ImGui_ImplOpenGL2_RenderDrawData ( ImGui::GetDrawData ( ) );
```

**Screenshots/Video**
![5p9g](https://user-images.githubusercontent.com/19976600/44261970-973c6580-a219-11e8-9eff-59eab9a6f68f.png)

**Comment by Zorrr** [comment]
Edit-- Error has been fixed, I was not drawing in my own OpenGL context

**Comment by Zorrr** [comment]
Issue has been resolved thanks to the Win32 OpenGL thread. My final code is this:
https://pastebin.com/YfGXuruQ

**Comment by ice1000** [comment]
#2022

**Comment by ocornut** [comment]
> Issue has been resolved thanks to the Win32 OpenGL thread.

@Zorrr Glad to hear. Could you clarified what exactly fixed it? (I recall #1459 also had weird possibly-similar rendering issues in some of the screenshots.)

**Comment by zorftw** [comment]
> > Issue has been resolved thanks to the Win32 OpenGL thread.
> 
> @Zorrr Glad to hear. Could you clarified what exactly fixed it? (I recall #1459 also had weird possibly-similar rendering issues in some of the screenshots.)

This might be too late to say, since we're already almost 2 years later. But the rendering relied in the OpenGL context. Creating a high-definition OpenGL context resolved the issues.
